draft-wang-ppm-differential-privacy.md

```diff
@@ -239,11 +254,133 @@ The following considerations are out-of-scope for this document:
 
 This document uses the same conventions for error handling as {{!DAP}}.
 
-> TODO: add more
+Let `exp(EPSILON)` denote raising the numeric constant `e` to the power of
+`EPSILON`.
+
+# Security Goals and Trust Model {#overview}
```

Comment:
Correct me if I'm wrong: I believe the goal of this section is to provide an overview of DP, including definitions (done in this PR), trust models (done in this PR), utility (to be filled) etc. Should the title of this section be changed to something like "Overview" as well? And things like utility can be added as a subsection in the overview.

Reply:
I think we can bring back some section from old overview once we merge this. For e.g. local-DP should be explained before it's mentioned in Hedging

Reply:
That could work. I was thinking of adding a new section to Trust Models, something like this:

```
## NOP: No-honest-parties

If no parties are honest, then the best we can hope for is that the honest
Clients' measurements have "local-DP". This property is defined the same way as
pure- or approximate-DP, except that the bound that we aim to achieve is looser
than what we can get when at least one Aggregator is honest (OAMC or OA).
```

Reply:
sounds good to me, maybe worth mentioning there is still at least one honest client worthy protecting for.

Reply:
Done.